tests/main/ubuntu-core-gadget-config-defaults/task.yaml

```diff
@@ -55,10 +59,14 @@ restore: |
         echo "This test needs test keys to be trusted"
         exit
     fi
-    echo "Undo the rsyslog disable"
-    systemctl unmask rsyslog.service || true
-    systemctl enable rsyslog.service || true
-    systemctl start rsyslog.service || true
+    echo "Undo the service disable"
+    if [ "$SERVICE" = ssh ]; then
+        rm -f /etc/ssh/sshd_not_to_be_run
```

Comment:
should we have a "start" here too for symmetry

Reply:
done